Dockerfile

```diff
@@ -0,0 +1,82 @@
+# syntax=docker/dockerfile:1
+
+ARG GO_VERSION=1.16.7
+ARG XX_VERSION=1.1.2
+ARG OSXCROSS_VERSION=11.3-r7-alpine
+
+ARG PKG=github.com/docker/docker-credential-helpers
+
+# xx is a helper for cross-compilation
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:${XX_VERSION} AS xx
+
+# osxcross contains the MacOSX cross toolchain for xx
+FROM crazymax/osxcross:${OSXCROSS_VERSION} AS osxcross
```

Comment:
This is built from https://github.com/crazy-max/osxcross ? Or from the upstream one? (I see neither repo tags releases, so was wondering what version it was built from)

Reply:
> This is built from https://github.com/crazy-max/osxcross ?

It's built from https://github.com/crazy-max/docker-osxcross